### ISSUE TITLE
got the nan for loss after training several epoches

### ISSUE THREAD

**Issue by zllrunning** [NieXC/pytorch-mula#1]
Hi, I trained the model, but after several epoch, it got the nan for loss.val and loss.avg. The only thing I do is to modify the batch size to 14. Then I reduce learning rate to 0.001, but the same situation has occurred. My puzzle is that the accuracy was rising and loss was normal before this situation suddenly occurs.

**Comment by NieXC** [comment]
@zllrunning 

I think this problem is due to the bug for batch normalization layer when using CUDNN. You can first try to disable the CUDNN for batch normalization layer or disable it for the whole network during training, which would help you to avoid the above problem.

You can also try to use Pytorch 0.2.0 for training and testing, which works normally.

The above two ways should help to solve the problem. If they do not help, we can have further discussion to find the reason.

**Comment by zllrunning** [comment]
Thanks! This problem has been solved by using Pytorch 0.2.0.

**Comment by zzw1123** [comment]
@NieXC Hi, I have the same 'NAN' problem but my pytorch version is already 0.2.0.
So could you please help me out?
